tests/api_tests/test_create_index.py

```diff
@@ -0,0 +1,39 @@
+from marqo.client import Client
+from marqo.errors import MarqoApiError, MarqoWebError
+import unittest
+import pprint
+from tests.marqo_test import MarqoTestCase
+
+
+class TestCreateIndex(MarqoTestCase):
+
+    def setUp(self) -> None:
+        client_0 = Client(**self.client_settings)
+        self.index_name_1 = "my-test-index-1"
+        for ix_name in ['some-bulk', self.index_name_1]:
```

Comment:
instead of hard coding, we could have 
```
self.index_name_bulk_substring = "some-bulk"
```

Then just reference this in the delete and creation code